modules/standard/IO.chpl

```diff
@@ -3863,7 +3863,7 @@ config param useNewLinesRegionBounds = false;
 
    :throws SystemError: Thrown if an object could not be returned.
  */
-deprecated "`file.lines` is deprecated; please use `fileReader.lines` instead"
+deprecated "`file.lines` is deprecated; please use `fileReader.lines` by calling `.reader().lines() on your `file`"
```

Comment:
I might go with the message "please use `file.reader().lines` instead" or something like that, but this is fine too.

Basically, as a user of this function, all I wanted was to know was "what do I need to change?" and I think just making that as clear as possible is the best way to go, but I suppose it is up for debate whose wording is more clear, so I am fine either way (and you definitely have more 2.0 knowledge than me, so it is likely that you know better than I what makes sense here).